cfgs/ladybug.cfg

```diff
@@ -0,0 +1,4 @@
+input_overlay = "/opt/retropie/emulators/retroarch/overlays/arcade-bezels/ladybug.cfg"
+input_overlay_enable = "true"
+input_overlay_opacity = "0.900000"
```

Comment:
I would put the opacity as 1.000000 to make it completely opaque, although it won't make any difference

Reply:
I'll look into that, too. Thanks!

Reply:
Done, see second commit.